services/scan/src/end_to_end.test.ts

```diff
@@ -42,27 +42,18 @@ afterEach(async () => {
 });
 
 test('going through the whole process works', async () => {
-  // Do this first so interpreter workers get initialized with the right value.
   await request(app)
-    .patch('/central-scanner/config/skipElectionHashCheck')
-    .send({ skipElectionHashCheck: true })
-    .set('Content-Type', 'application/json')
+    .patch('/central-scanner/config/election')
+    .send(asElectionDefinition(election).electionData)
+    .set('Content-Type', 'application/octet-stream')
     .set('Accept', 'application/json')
     .expect(200, { status: 'ok' });
 
-  {
-    // try export before configure
-    const response = await request(app)
-      .post('/central-scanner/scan/export')
-      .set('Accept', 'application/json')
-      .expect(200);
-    expect(response.text).toBe('');
-  }
-
```

Comment:
Why would we test export before configure?

Reply:
If we are concerned about this, I think we should probably `400` on this.

Reply:
Agreed it should be 400. And why test? To make sure it does send the right response code.